### PR TITLE
Added an Ansys CLI section

### PR DESCRIPTION
**Pull request by jhodrien**
It's fairly bare, but then I know almost nothing about running this.  I was hoping that by having a section we might spur on some people to give it a try and tell us how it should be used.

### CODE REVIEW

File: book/software/applications/ansys/ansyscli.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+```bash
+$ module add ansys/2020R2
+$ ansys202 -g
+```
+
+This runs it graphically, so it's important you have X forwarding enabled or are using [X2Go](../../../getting_started/x2go).
+
+### Running through the batch queues
+
+When running through the [batch queues](../../../usage/batchjob), no interactive input is possible. It is possible to create a journal file which contains all of the commands that would normally be entered within Ansys CLI, in a similar vein to how it works with Fluent.
```
Comment (Sparrow0hawk) [inline book/software/applications/ansys/ansyscli.md:31]:
```suggestion
When running through the [batch queues](../../../usage/batchjob), no interactive input is possible. It is possible to create a journal file which contains all of the commands that would normally be entered within Ansys CLI, in a similar way to how it works with Fluent.
```
*Opens debate about whether "similar vein" is as accessible to people for whom english isn't a first language

File: book/software/applications/ansys/ansyscli.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+It is then necessary to construct a job submission script that will run Ansys CLI. The job submission script begins with a request for resources and the executable to be run, e.g.:
```
Comment (Sparrow0hawk) [inline book/software/applications/ansys/ansyscli.md:33]:
```suggestion
It is then necessary to construct a [job submission script](../../../usage/batchjob.html#resource-specification) that will run Ansys CLI. The job submission script begins with a request for resources and the executable to be run, e.g.:
```

File: book/software/applications/ansys/ansyscli.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+ansys202 -np $NSLOTS -acc nvidia -p ANSYS -b -i example.inp -o example.out
+```
+
+The file can be submitted to the queue by typing this (assuming you'd written the above into a file called ansys.sh):
+
+```bash
+$ qsub ansys.sh
+```
+
+You'd really want to make sure that this gave you a significant performance advantage, and please do let us know how you get on with this if you do experiment with GPU acceleration.
```
Comment (Sparrow0hawk) [inline book/software/applications/ansys/ansyscli.md:110]: Maybe turn this into an admonition?

**Review by Sparrow0hawk — CHANGES_REQUESTED** [review]
One typo but other comments are just suggestions so happy to approve once resolved.

File: book/software/applications/ansys/ansyscli.md (hunk 3 above)
Comment (jhodrien) [reply book/software/applications/ansys/ansyscli.md]: Done.

**Review by Sparrow0hawk — APPROVED** [review]

**Pull request merged**